src/Routing/RouteBuilder.php

```diff
@@ -215,6 +215,11 @@ public function addExtensions($extensions)
      */
     public function path(): string
     {
+        $routeKey = strpos($this->_path, '{');
+        if ($routeKey !== false && strpos($this->_path, '}') !== false) {
```

Comment:
Shouldn't this look for `}` after `{`?

Reply:
That's what's it's doing.

Reply:
I mean after the position in the string.

Reply:
Ah, ideally yes. I copied the check from another place in the routing code. Guess I can update all, though if one has a closing brace before an opening brace then their route template is most likely messed up.

Reply:
Up to you if you want to throw a validation error if } is before {.

Reply:
I don't think such a check is necessary. A broken route would be indication enough. Being over zealous in handling dev error didn't go too well last time we tried :wink:.